chart-parts/ingress.yaml

```diff
@@ -14,22 +14,27 @@ apiVersion: "v1"
 kind: "Secret"
 type: kubernetes.io/tls
 metadata:
-  name: "{{ .Release.Namespace }}-secrets-nic"
+  name: "{{ .Values.services.ingress.class }}-ingress-tls"
+  namespace: "{{ .Release.Namespace }}"
 data:
   tls.crt: ""
   tls.key: ""
 ---
 apiVersion: "extensions/v1beta1"
 kind: "Ingress"
 metadata:
-  name: "ingress-uaa"
-  namespace: {{ .Release.Namespace | quote }}
+  name: "{{ .Release.Name }}-{{ .Values.services.ingress.class }}"
+  namespace: "{{ .Release.Namespace }}"
```

Comment:
Note that use of `| quote` is pretty standard for us to ensure proper (non)quoting of data.
See pretty much all the generated templates.
What does going to `"..."` bring us ?
(Would prefer consistency with everything else)

The name changes look ok

Reply:
The `quote` function does nothing more than wrapping the value on quotes, as we can see the implementation https://github.com/Masterminds/sprig/blob/b1fe2752acccf8c3d7f8a1e7c75c7ae7d83a1975/strings.go#L103.

`.Release.Namespace` could even be used without quotes here, since it's a valid string (and cannot ever be something else). The use of `"{{ ... }}"` here is merely a formality. If you prefer, I can get rid of the quotes here and below in the places there it makes sense.

Reply:
I guess no need to. I will accept as is.

Reply:
> The `quote` function does nothing more than wrapping the value on quotes

This is not quite true: it will also escape any embedded quotes inside the string and makes sure the result is valid Go syntax (it used `%q` instead of `%s` to print the string).

So it is generally preferable to making assumption of the characters that you are going to print.

Reply:
Thanks for pointing that out @jandubois. I'll keep that in mind and use it when appropriate. In this case we don't need it (quotes are not valid characters for namespaces).